GlucoseOnFhir/GlucoseMeterViewController.swift

```diff
@@ -203,12 +290,16 @@ class GlucoseMeterViewController: UITableViewController, GlucoseProtocol, FHIRPr
     
     override func tableView(_ tableView: UITableView, titleForHeaderInSection section: Int) -> String? {
         switch section {
-            case 0:
-                return "Glucose Meter Features"
-            case 1:
-                return "Glucose Record Count"
-            case 2:
-                return "Glucose Records"
+            case Section.section0.rawValue:
+                return Section.section0.description()
```

Comment:
Again, no need for a switch. create a Section enum and then just call `description()`.